src/main/java/io/cdap/plugin/servicenow/sink/transform/ServiceNowRecordWriter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.servicenow.sink.transform;
+
+import com.google.gson.JsonObject;
+import io.cdap.plugin.servicenow.sink.ServiceNowSinkConfig;
+import io.cdap.plugin.servicenow.sink.model.RestRequest;
+import io.cdap.plugin.servicenow.sink.service.ServiceNowSinkAPIRequestImpl;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapreduce.RecordWriter;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *  ServiceNow Record Writer class to insert/update records
+ */
+public class ServiceNowRecordWriter extends RecordWriter<NullWritable, JsonObject> {
+
+  private ServiceNowSinkConfig config;
+  private List<RestRequest> restRequests = new ArrayList<>();
+  private ServiceNowSinkAPIRequestImpl servicenowSinkAPIImpl;
+
+  private static final Logger LOG = LoggerFactory.getLogger(ServiceNowRecordWriter.class);
+
+  public ServiceNowRecordWriter(ServiceNowSinkConfig config) {
+    super();
```

Comment:
We can skip super call.

Reply:
Skipped. f52f02d4f710d02cead2bed152cda9004a27c735